docs/components/core/Providers.tsx

```diff
@@ -1,13 +1,18 @@
 import * as React from 'react'
 
-import { Provider, configureChains, createClient, defaultChains } from 'wagmi'
+import {
+  WagmiProvider,
+  configureChains,
+  createClient,
+  defaultChains,
+} from 'wagmi'
 
 import { CoinbaseWalletConnector } from 'wagmi/connectors/coinbaseWallet'
 import { InjectedConnector } from 'wagmi/connectors/injected'
 import { MetaMaskConnector } from 'wagmi/connectors/metaMask'
 import { WalletConnectConnector } from 'wagmi/connectors/walletConnect'
 
-import { alchemyProvider } from '../../../packages/core/apiProviders/alchemy'
+import { alchemyProvider } from '../../../packages/core/providers/alchemy'
```

Comment:
Can we import this from the wagmi package entrypoint?

Reply:
hmm, tried to do this, but it was yelling at me

Reply:
fixed now